ops/solana/solana.go

```diff
@@ -331,6 +331,24 @@ func (d Deployer) InitOCR(keys []opsChainlink.NodeKeys) error {
 
 	fmt.Println("Setting up OCR Feed:")
 
+	fmt.Println("Setting Billing...")
+	input := map[string]interface{}{
+		"observationPaymentGjuels":  1,
+		"transmissionPaymentGjuels": 1,
+	}
+	jsonInput, err := json.Marshal(input)
+	if err != nil {
+		return err
+	}
```

Comment:
we could but i think having a map[string] makes it easier in the future if additional parameters get added. it feels easier to read too rather than a single line string

Reply:
Ah, it's debatable :) But makes sense to leave it for now, and not change just this one instance.